zubhub_frontend/zubhub/src/components/form/tagsInput/TagsInput.jsx

```diff
@@ -31,7 +32,7 @@ export default function TagsInput({
   const classes = makeStyles(tagsInputStyles)();
   const [isSearching, setIsSearching] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
-  const selectedTags = [...(initialSelectedTags ?? [])];
+  const selectedTags = useMemo(() => [...(initialSelectedTags ?? [])], [initialSelectedTags]);
```

Comment:
is this necessary for our upgrade to work?

Reply:
Not really, but very straight forward.  This can't delay anything